plutus-tx/src/PlutusTx/IsData/Class.hs

```diff
@@ -46,6 +46,7 @@ class UnsafeFromData (a :: Type) where
     --
     -- When implementing this function, make sure to call 'unsafeFromBuiltinData'
     -- rather than 'fromBuiltinData' when converting substructures!
+    -- This is a simple type without any validation, __use with caution__.
```

Comment:
nitpicking: I don't know if your intentions is to have a newline but you need an empty comment line to have a newline. Right now, it is concatenated with the text above.

Reply:
I didn't have that intention but that's a good idea.